src/components/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+// import { useParams } from "react-router-dom";
+import { getArticles} from "./api";
+
+const Articles = () => {
+    const [articleList, setArticleList] = useState([])
+    // const { article_name } = useParams();
+    
+    useEffect(() => {
+        console.log('hello')
+        getArticles().then(({data}) => {
+            console.log(data);
+            setArticleList(data);
+        });
+    
```

Comment:
Should have a loading pattern here, so that while the articles are loading, the user sees something other than a blank space. If you're not sure how to do this, check out the bottom of the useEffect notes https://notes.northcoders.com/courses/js-front-end/useEffect 🙂